File: ouweh-vdwerf/src/main/org/uva/ql/ast/expression/binary/And.java
package main.org.uva.ql.ast.expression.binary;

import main.org.uva.ql.ast.expression.Expression;

public class And extends BinaryOperation {

    public And(Expression left, Expression right){
        super(left,right);
    }

    @Override
    public String toString() {
        return String.format("(%s AND %s)", this.getLeft(), this.getRight());
    }
}

File: Erik_Vincent/Assignment1/Assignment1/Model/QL/AST/Expression/And.cs
namespace Assignment1.Model.QL.AST.Expression
{
    public class And : Binary
    {
        public And(int lineNumber, IExpression left, IExpression right) : base(left, right)
        {
            _lineNumber = lineNumber;
        }

        public override void Accept(IExpressionVisitor visitor) => visitor.Visit(this);
    }
}